src/lexer.jl

```diff
@@ -27,8 +27,9 @@ mutable struct Closer
 end
 Closer() = Closer(true, true, false, false, false, false, false, false, false, false, false, false, false, false, false, false, false, false, -1)
 
+const IOT = typeof(IOBuffer().data)
 mutable struct ParseState
-    l::Lexer{Base.GenericIOBuffer{Array{UInt8,1}},RawToken}
+    l::Lexer{Base.GenericIOBuffer{IOT},RawToken}
```

Comment:
Apparently not 🤷. I didn't do that because I am not sure all the places that store this state and would need another type parameter for it to keep being concrete.

Reply:
Fair enough.